### ISSUE TITLE
Accuracy on ImageNet21K-P

### ISSUE THREAD

**Issue by imankgoyal** [Alibaba-MIIL/ImageNet21K#2]
Hello,

Great work! Thanks for sharing. I was wondering if you can please share the training set accuracy and curves on ImageNet21K-P. In the appendix, I think you have reported the final validation set accuracy. It would be great if you can also share the final accuracy and training curves on the train set. It would help to debug any issue with other models.

Best,
Ankit

**Comment by mrT23** [comment]
Hi Ankit

notice that we provide in the repo a full training reproduction script:
https://github.com/Alibaba-MIIL/ImageNet21K/blob/main/train_semantic_softmax.py
i tested it, and it reached the article's accuracy.
i am not calculation training metrics in it, but you can easily add them.

its not a bad idea to add the training loss curves, but my original runs were without tensorboard, so I would have to send runs again to get them. might do that in the future.
